README.md

```diff
@@ -16,14 +48,61 @@ Python tools for photovoltaic degradation analysis.
 import rdtools
 ```
 
-For usage examples, look at the notebooks in rdtools/docs.
+The most frequently used functions are:
+
+```Python
```

Comment:
For all the functions I suggest referencing the module they come from, for example `normalization.normalize_with_pvwatts(energy, pvwatts_kws)`. Even thought they can be called without the module name, this will help people find the documentation and source code if they want to dig in.

Reply:
OK